portfolio/src/main/webapp/script.js

```diff
@@ -29,6 +29,27 @@ function addRandomShow() {
 
 async function getName() {
   const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('hello-container').innerText = quote;
+  const name = await response.text();
+  document.getElementById('hello-container').innerText = name;
+}
+function getDataStats() {
+  fetch('/data').then(response => response.json()).then((stats) => {
```

Comment:
For consistency, I would use `await` here as well, or alternatively use promise syntax in `getName`. Either works, but it's easier to read code that uses the same style everywhere :slightly_smiling_face: